recipes/resource_pool/all/conanfile.py

```diff
@@ -37,44 +35,55 @@ def _compilers_minimum_version(self):
             "apple-clang": "10",
         }
 
-    def _validate_compiler_settings(self):
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("boost/[<=1.79]")
```

Comment:
We are still not using version ranges for anything other than CMake and OpenSSL - I'm surprised this one does not work with boost 1.80 or greater . I'd suggest keeping this on 1.79 if that's the highest it supports for now.

```suggestion
        self.requires("boost/1.79.0")
```